utils/bootstrap.go

```diff
@@ -45,6 +45,23 @@ func BootStrap(
 	errors []*RosettaTypes.Error,
 	client construction.Client,
 ) error {
+	err := cfg.Validate()
+	if err != nil {
+		return err
+	}
+
+	logger, syncFn, err := stats.InitLogger(cfg)
+	if err != nil {
+		return err
+	}
+	defer syncFn()
+
+	statsdClient, done, err := stats.InitStatsd(logger, cfg, "c3/rosetta-api")
```

Comment:
we might want to make this as env variable too.

Reply:
Yup, will update this

Reply:
Done